### PR TITLE
PLI dilution for 0.x. Fixes-according-to-PR-3422-remark

### PR DESCRIPTION
**Pull request by natikaltura**
Hi [@lminiero](https://github.com/lminiero),

I had some git issues, so I created this new PR (and branch) for the 0.x patch.

-	The original PR and our discussion is in  : https://github.com/meetecho/janus-gateway/pull/3422

-	I've updated the multistream patch in the original PR. Please review. https://github.com/meetecho/janus-gateway/pull/3423

Thanks

### CODE REVIEW

File: rtp.c [hunk 1]
```diff
@@ -1173,8 +1174,8 @@ gboolean janus_rtp_simulcasting_context_process_rtp(janus_rtp_simulcasting_conte
 					if(context->substream_target_temp < 0)
 						context->substream_target_temp = 0;
 					if(context->substream_target_temp != prev_target) {
-						JANUS_LOG(LOG_WARN, "No packet received on substream %d for a while, falling back to %d\n",
-							context->substream, context->substream_target_temp);
+						JANUS_LOG(LOG_WARN, "No packet received on substream %d for %d Milliseconds, falling back to %d\n",
```
Comment (tmatth) [inline rtp.c:1177]: You may want `PRId64` instead of `%d` here

Comment (lminiero) [reply rtp.c:1177]: Our code style is to use `%"SCNi64"` in those instances (assuming this is a `int64_t`?).

Comment (tmatth) [reply rtp.c:1177]: Ah ok, that makes since given that internally `janus_vprintf` is calling `g_vasprintf`.

Comment (natikaltura) [reply rtp.c]: Hi, Thanks. I've fixed it. Please also review the multistream PLI fix; it includes additional changes: https://github.com/meetecho/janus-gateway/pull/3423."

**Comment by natikaltura** [comment]
"Hi, I've fixed it. Please also review the multistream PLI fix; it includes additional changes: https://github.com/meetecho/janus-gateway/pull/3423."

**Comment by lminiero** [comment]
See https://github.com/meetecho/janus-gateway/pull/3423#issuecomment-2470159191

**Comment by natikaltura** [comment]
Hi @lminiero, thanks for your comments! All fixed and pushed.

**Comment by lminiero** [comment]
Merging :muscle:

**Pull request merged**